server/channels/app/expirynotify.go

```diff
@@ -24,6 +24,12 @@ func (a *App) NotifySessionsExpired() error {
 	// Get all mobile sessions that expired within the last hour.
 	sessions, err := a.ch.srv.Store().Session().GetSessionsExpired(OneHourMillis, true, true)
 	if err != nil {
+		a.NotificationsLog().Error("Cannot get sessions expired",
+			mlog.String("type", TypePush),
+			mlog.String("status", StatusServerError),
+			mlog.String("reason", ReasonFetchError),
+			mlog.Err(err),
+		)
 		return model.NewAppError("NotifySessionsExpired", "app.session.analytics_session_count.app_error", nil, "", http.StatusInternalServerError).Wrap(err)
```

Comment:
Same as before. Logging and returning an error is not a good pattern. Either we should log at the call site, or probably what we really want is metrics.